.travis.yml

```diff
@@ -1,20 +1,15 @@
 language: python
+sudo: false
 python:
   - "2.6"
 services:
-  - elasticsearch
   - redis-server
 before_install:
-  - sudo apt-get update
-  - sudo apt-get install swig pngcrush
```

Comment:
For posterity:  We decided that mocking pngcrush was more appropriate for our tests.  Thanks to mat for the patch.